### PR TITLE
Update toolbox to support codecov diff.

### PR DESCRIPTION
**Pull request by hklai**
1. Adding a way to get base SHA from an existing PR in githubctl.

2. Updating pkg_check to generate diff from two coverage files.

### CODE REVIEW

File: toolbox/githubctl/main.go [hunk 1]
```diff
@@ -291,8 +301,10 @@ func readPostsubmitListFromProwConfig(org, repo, branch string) map[string]struc
 
 func init() {
 	flag.Parse()
+	var token string
```
Comment (rkpagadala) [inline toolbox/githubctl/main.go:304]: Why these changes? Using := to declare is also idiomatic go

**Review by rkpagadala — APPROVED** [review]
/lgtm

File: toolbox/githubctl/main.go (hunk 1 above)
Comment (hklai) [reply toolbox/githubctl/main.go:304]: oh it was due to some previous change when token and error could have been assigned different. I had since removed that code but did not convert this back.
